jenkins/catapult/jenkins/catapult-client-build-compiler-image-all.groovy

```diff
@@ -33,58 +46,68 @@ pipeline {
 				stage('gcc prior') {
 					steps {
 						script {
-							dispatchBuildCompilerImageJob('gcc-prior', 'ubuntu')
+							dispatchBuildCompilerImageJob('gcc-prior', 'ubuntu', "${ARCHITECTURE}")
 						}
 					}
 				}
 				stage('gcc latest') {
 					steps {
 						script {
-							dispatchBuildCompilerImageJob('gcc-latest', 'ubuntu')
+							dispatchBuildCompilerImageJob('gcc-latest', 'ubuntu', "${ARCHITECTURE}")
 						}
 					}
 				}
 				stage('gcc [debian]') {
 					steps {
 						script {
-							dispatchBuildCompilerImageJob('gcc-debian', 'debian')
+							dispatchBuildCompilerImageJob('gcc-debian', 'debian', "${ARCHITECTURE}")
 						}
 					}
 				}
 				stage('gcc [fedora]') {
 					steps {
 						script {
-							dispatchBuildCompilerImageJob('gcc-latest', 'fedora')
+							dispatchBuildCompilerImageJob('gcc-latest', 'fedora', "${ARCHITECTURE}")
 						}
 					}
 				}
 
 				stage('clang prior') {
 					steps {
 						script {
-							dispatchBuildCompilerImageJob('clang-prior', 'ubuntu')
+							dispatchBuildCompilerImageJob('clang-prior', 'ubuntu', "${ARCHITECTURE}")
 						}
 					}
 				}
 				stage('clang latest') {
 					steps {
 						script {
-							dispatchBuildCompilerImageJob('clang-latest', 'ubuntu')
+							dispatchBuildCompilerImageJob('clang-latest', 'ubuntu', "${ARCHITECTURE}")
 						}
 					}
 				}
 
 				stage('msvc latest') {
```

Comment:
should you make these amd64 only tasks conditional on arch in daily and weekly workflows, like in image workflows?

Reply:
yea, my thoughts behind tests were to run all the jobs independent of architecture.  So we ran mainly arm64 tests during the week, the sans tests will still get executed.  Currently the default is still run jobs on amd64.